cn/lessons/basics/mix.md

```diff
@@ -0,0 +1,146 @@
+---
+
+layout: page title: Mix category: basics order: 8
```

Comment:
Two issues I found in `mix.md` and `pattern-matching.md` just to be aware of:

+ Newlines were stripped from meta data
+ Metadata block must be start and end with exactly three dashes `---`